### ISSUE TITLE
something about dropout

### ISSUE THREAD

**Issue by douyh** [pytorch/pytorch#13659]
### Please note that this issue tracker is not a help form and this issue will be closed.

When I use nn.Dropout() in my network, I am confused by the ouput of training loss and validation loss.
I use```self.drop = nn.Dropout(0.7)```  when I initialize my network, and I add ```x = self.drop(x)``` between 2 fc layers.
However, when I print my loss, the loss on validation set is MUCH HIGHER than training loss from the FIRST BATCH. It seems to be divided by a factor 0.3.
In the first 100 batches, my training loss is 4.7, at the same time, my validating loss is 15.1.
What happened?
I use ```model.eva()``` before test on validation set and ```model.train()``` before training.

**Comment by soumith** [comment]
discuss.pytorch.org
